containerbuddy/memcached.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
is there a reason to be inconsistent in the naming?

- `reload-db.sh`
- `memcached.sh`